src/components/containers/popover/PopoverLink.js

```diff
@@ -65,7 +67,8 @@ class PopoverLink extends React.Component {
           }}
         >
           <PopoverButton
-            aria-haspopup="dialog"
+            aria-haspopup="true"
```

Comment:
This stuff is so confusing. :)  I think you are right, we use role and aria-labelledby, and probably don't use aria-haspopup.